src/DialogMessage.ts

```diff
@@ -23,6 +23,7 @@ export namespace DialogMessage {
     export const invalidWebappFolder: string = localize('tomcatExt.invalidWebappFolder', 'The folder is not a valid web app to run on Tomcat Server.');
     export const invalidWarFile: string = localize('tomcatExt.invalidWarFile', 'Please select a .war file.');
     export const pickFolderToGenerateWar: string = localize('tomcatExt.pickFolderToGenerateWar', 'Please select the folder(s) you want to generate war package');
+    export const serverAlreadyAdded: string = localize('tomcatExt.serverAlreadyAdded', 'That server has already been added to the list.');
```

Comment:
New error condition when using the Run In Place feature. It was previously impossible to have two server entries pointing to the same CATALINA_BASE.